tests/cases/rect.py

```diff
@@ -1,12 +1,16 @@
+"""This file implements a rectangle class """
+
 from dataclasses import dataclass
 
 
 @dataclass
 class Rectangle:
+    "Rectangle as a dataclass"
```

Comment:
can we use triple docstrings, per [PEP8](https://www.python.org/dev/peps/pep-0008/#id33)/[PEP257](https://www.python.org/dev/peps/pep-0257/)